Dockerfile

```diff
@@ -2,8 +2,8 @@ ARG BASE_VARIANT=alpine
 ARG GO_VERSION=1.17
 ARG XX_VERSION=1.1.0
```

Comment:
There is a newer version which seems to solve some MacOS issues: https://github.com/tonistiigi/xx/releases/tag/v1.1.2